Resources/migration.yml

```diff
@@ -386,9 +386,7 @@ ClothingBeltSuspenders: ClothingBeltSuspendersRed
 ClothingNeckShockCollar: ClothingBackpackElectropack
 
 # 2024-08-22
-ComputerShuttleSalvage: null
-SalvageShuttleConsoleCircuitboard: null
-SalvageShuttleCircuitboardStealObjective: null
```

Comment:
Just comment instead of deleting them. However, the maps have likely been re-saved upstream since this change so you will need to make map patches for upstream maps and ensure they are mapped on any CD maps.

Reply:
I am actually not sure that map patches are needed, it might be better to not include the console on the station to drive home the fact it is really run down.

Reply:
I've made them commented, should be good now.